jerry-core/ecma/base/ecma-alloc.h

```diff
@@ -38,16 +38,16 @@ extern ecma_object_t *ecma_alloc_object (void);
 extern void ecma_dealloc_object (ecma_object_t *);
 
 /**
- * Allocate memory for ecma-property
+ * Allocate memory for ecma-property pair
  *
  * @return pointer to allocated memory
  */
-extern ecma_property_t *ecma_alloc_property (void);
+extern ecma_property_pair_t *ecma_alloc_property_pair (void);
 
 /**
- * Dealloc memory from an ecma-property
+ * Dealloc memory from an ecma-property pair
  */
-extern void ecma_dealloc_property (ecma_property_t *);
+extern void ecma_dealloc_property_pair (ecma_property_pair_t *);
```

Comment:
Please, move both these declarations to the end of the header file to match the order of functions in the C source.